fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java

```diff
@@ -0,0 +1,154 @@
+        super(PlanType.SHOW_REPLICA_STATUS_COMMAND);
+        this.tableRefInfo = tableRefInfo;
+        this.where = where;
+    }
+
+    @Override
+    public ShowResultSet doRun(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        validate(ctx);
+        List<List<String>> results;
+        try {
+            results = MetadataViewer.getTabletStatus(tableRefInfo.getTableNameInfo().getDb(),
+                tableRefInfo.getTableNameInfo().getTbl(),
+                tableRefInfo.getPartitionNamesInfo().getPartitionNames(), statusFilter, where);
+        } catch (DdlException e) {
+            throw new AnalysisException(e.getMessage());
+        }
+        return new ShowResultSet(getMetaData(), results);
+    }
+
+    public ShowResultSetMetaData getMetaData() {
+        ShowResultSetMetaData.Builder builder = ShowResultSetMetaData.builder();
+        for (String title : TITLE_NAMES) {
+            builder.addColumn(new Column(title, ScalarType.createVarchar(30)));
+        }
+        return builder.build();
+    }
+
+    /**
+     * validate for show replica
+     */
+    public void validate(ConnectContext ctx) throws AnalysisException, UserException {
+        // check auth
+        if (!Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
+        }
+
+        // bind relation
+        tableRefInfo.analyze(ctx);
+
+        if (!validateWhere()) {
+            throw new AnalysisException(
+                "Where clause should looks like: status =/!= 'OK/DEAD/VERSION_ERROR/SCHEMA_ERROR/MISSING'");
+        }
+    }
+
+    private boolean validateWhere() throws AnalysisException {
+        // analyze where clause if not null
+        if (where == null) {
+            return true;
+        }
+
+        if (!(where instanceof EqualTo || (where instanceof Not && where.child(0) instanceof EqualTo))) {
+            return false;
+        }
+
+        EqualTo equalTo = where instanceof EqualTo ? (EqualTo) where : (EqualTo) ((Not) where).child();
+
+        Expression leftChild = equalTo.child(0);
+        if (!(leftChild instanceof UnboundSlot)) {
+            return false;
+        }
+
+        String leftKey = ((UnboundSlot) leftChild).getName();
+        if (!leftKey.equalsIgnoreCase("status")) {
+            return false;
+        }
+
+        Expression rightChild = equalTo.child(1);
+        if (!(rightChild instanceof StringLikeLiteral)) {
+            return false;
+        }
+
+        try {
+            statusFilter = Replica.ReplicaStatus
+                    .valueOf(((StringLikeLiteral) rightChild).getStringValue().toUpperCase());
+        } catch (Exception e) {
+            return false;
+        }
+
+        if (statusFilter == null) {
+            return false;
+        }
+
+        return true;
+    }
+
```

Comment:
override checkSupportedInCloudMode method and implement as checkStmtSupported() method in ShowExecutor.java

Reply:
done